File: modul 2/PRAK202-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>

int main(){
    float NP, NK, hasil;;
    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &NP);
    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &NK);
    hasil = NP + NK;
    if(NP==(int)NP){
        printf("Hasil dari penjumlahan nilai pertama \"%d\" dan nilai kedua\"%.1f\" adalah \"%.2f\"\n\n", (int)NP,NK,hasil);
    }
    else{
        printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua\"%.1f\" adalah \"%.2f\"\n\n", NP,NK,hasil);
    }
    return 0;
}

File: modul 2/PRAK202-2410817320004-AmandhaCitraMustika.py
NP = float(input("Masukkan Nilai Pertama : "))
NK = float(input("Masukkan Nilai Kedua : "))
hasil = NP + NK

if NP.is_integer():
    print(f"Hasil dari penjumlahan nilai pertama \"{int(NP)}\" dan nillai kedua \"{NK: .1f}\" adalah \"{hasil: .2f}\"")
else : 
    print(f"Hasil dari penjumlahan nilai pertama \"{NP: .2f}\" dan nilai kedua \"{NK: .1f}\" adalah \"{hasil: .2f}\"")